neo.UnitTests/UT_Block.cs

```diff
@@ -214,12 +215,12 @@ public void ToJson()
 
             JObject jObj = uut.ToJson();
             jObj.Should().NotBeNull();
-            jObj["hash"].AsString().Should().Be("0x1d8642796276c8ce3c5c03b8984a1b593d99b49a63d830bb06f800b8c953be77");
+            jObj["hash"].AsString().Should().Be("0xcfe6769c50c9bf998f81b43fad967ca2858c95dbed248bc4926bbf7e36f88016");
             jObj["size"].AsNumber().Should().Be(161);
             jObj["version"].AsNumber().Should().Be(0);
             jObj["previousblockhash"].AsString().Should().Be("0x0000000000000000000000000000000000000000000000000000000000000000");
             jObj["merkleroot"].AsString().Should().Be("0xd841af3d6bd7adb4bca24306725f9aec363edb10de3cafc5f8cca948d7b0290f");
-            jObj["time"].AsNumber().Should().Be(4244941696);
+            jObj["time"].AsNumber().Should().Be(1514007552);
```

Comment:
What is this date?
Not that 4244941696 is reasonable either hehe

Reply:
I believe it fixed now.